web/src/message_edit.js

```diff
@@ -1069,7 +1069,10 @@ export function save_message_row_edit($row) {
                 }
 
                 hide_message_edit_spinner($row);
-                const message = channel.xhr_error_message("", xhr);
+                let message = channel.xhr_error_message("", xhr);
+                if (message === "") {
+                    message = "An unknown error occurred.";
```

Comment:
I think we might want to instead make `xhr_error_message` return a string like this for "500" or other unexpected error codes? 

Also, never add a string without translation tags.

Reply:
Discussed on CZO